docs/publishers/config.rst

```diff
@@ -288,6 +288,37 @@ loads.
     of that text when it is being viewed as well as the font-family of the
     editor as the annotation is being written.
 
+.. option:: onLayoutChange
+
+  ``function``. This function will be a registered callback to be invoked when the sidebar
+  layout changes. Changes to the layout occur on load, when the sidebar is toggled to
+  show and hide, and when the user adjusts the sidebar manually.
+
+  This setting can only be set using :js:func:`window.hypothesisConfig`.
+
+  When a layout change happens the registered :option:`onLayoutChange` function will
+  receive a single ``Object`` as it's argument. This object details the layout parameters
+  after the change.
+
+  Layout object available fields: 
+
+  .. option:: expanded
+
+    ``Boolean``. If the sidebar has gone into a state where it will not be usable by
```

Comment:
The wording here is a bit awkward and describing it as "usable" may be ambiguous, since the toolbar is still accessible when the sidebar is closed. Assuming that the reader is familiar with how to use the client as a user, I think it would suffice to just say something like "true if the sidebar is currently open".